src/components/Section/Relationships/RelationshipStatus/Marital.test.jsx

```diff
@@ -62,4 +63,197 @@ describe('The relationship status component', () => {
     expect(updates).toBe(2)
     expect(component.find('.accordion').length).toBe(1)
   })
+
+  describe('handles civil union dates', () => {
+    it('with good data', () => {
+      const props = {
```

Comment:
Since each of these props declarations are essentially the same with 1 different date object, maybe we can abstract the common props into one object, and have the test make a new object with just the new data, e.g.:

```
{ ...originalProps, CivilUnion: { EnteredCivilUnion: { ...otherProps }} }
```